docs/book/content/contributing/contributor_guide.md

```diff
@@ -95,10 +74,20 @@ section introduces some standard Git practices and guides you through
 the contribution process.
 
 
-## [Workflow](#workflow_link)
+(Sec_Workflow)=
+## Workflow
+
+(Sec_GitHubIssue)=
+### Submitting a GitHub Issue
+
+GitHub "issues" are an excellent way to ask questions, include code examples, and tag specific GitHub users. [TODO: Add more on GitHub issues]
```

Comment:
Unless it is specific to OG-USA, it's probably better to reference the `Git-Tutorial` repo so that we aren't duplicating any more than necessary across PSL projects.

Reply:
@jdebacker . I fully agree with this comment. But the following two considerations make me think we should leave this section as it stands for now.
1. I didn't really add much to this section, I just reformatted most of the points (with the exception of the "Submitting a GitHub Issue" section). We really need to update everything here from the "Setup Git" section and below. We should be referencing all of that to the [Git-Tutorial](https://pslmodels.github.io/Git-Tutorial/) documentation.
2. The [Git-Tutorial](https://pslmodels.github.io/Git-Tutorial/) documentation is not up to date. I think as soon as we have what we want from this section in the Git-Tutorial, then we rewrite this section of the OG-USA contributor guide to point to the Git-Tutorial.

So for now, I recommend that we leave this as it is.

Reply:
@jdebacker. Per your comment below, I have removed the "TODO" comment about adding more GitHub issues.